airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/containers.py

```diff
@@ -76,6 +76,7 @@ def format_java_container(dagger_client: dagger.Client) -> dagger.Container:
             "yum install -y findutils",  # gradle requires xargs, which is shipped in findutils.
             "yum clean all",
         ],
+        env_vars={"RUN_IN_AIRBYTE_CI": "1"},
```

Comment:
💅  I would love if we could add this to all containers. I know thats too large of a refactor for this though.

Reply:
> we could add this to all containers

I'd love this too, but maybe not all containers... Do we really want connector container we build in airbyte-ci to have this env var set?

Reply:
> Do we really want connector container we build in airbyte-ci to have this env var set?

Yes, we do, in fact it _needs_ to be set for connector builds. It's for the S3 build cache.

> I would love if we could add this to all containers. I know thats too large of a refactor for this though.

I thought this was going to be done as follow-up work to the formatting changes, we should be using the gradle base container in `gradle.py` when doing formatting tasks. I may have misunderstood your roadmap.

Reply:
We can do it for all java connectors easily and in the context of the follow-up work on formatting changes.
But I really meant all dagger containers, not only connectors, which would require a significant change in our logic.